scripts/modules/handlePageDetails.js

```diff
@@ -1,31 +1,36 @@
 import isNonEmptyString from './lang/isNonEmptyString.js';
-
-const MAP = {
-    ca: { country: 'CA', language: 'en' },
-    ca_fr: { country: 'CA', language: 'fr' },
-    en: { country: 'US', language: 'en' },
-};
+import localeMAP from './localeMAP.js';
 
 /**
  * Parses the page URL and retrieves the locale, cloud and category.
  * Page details are updated based on these values.
  */
 export default function handlePageDetails() {
-    const [locale, cloud, category] = window.location.pathname
+    const paths = window.location.pathname.replace('/bench', '')
```

Comment:
This could be overthinking it, but should we replace `/bench/` instead? I'm thinking maybe we get a `benchmark` category or something like that and it might interfere with this.

Reply:
we can remove it as soon as we finish the performance testing